lib/matplotlib/table.py

```diff
@@ -57,24 +60,40 @@ class Cell(Rectangle):
     fontproperties : dict
         A dict defining the font properties of the text. Supported keys and
         values are the keyword arguments accepted by `.FontProperties`.
+    visible_edges : str
+        A substring of 'BRTL' (bottom, right, top, left) or one
+        of {'left', 'center', 'right'}
     """
 
     PAD = 0.1
     """Padding between text and rectangle."""
 
+    _edges = 'BRTL'
+    """default value for visible edges."""
+
+    _edge_aliases = {'open':         '',
+                     'closed':       'BRTL',
+                     'horizontal':   'BT',
+                     'vertical':     'RL'
+                     }
+
     def __init__(self, xy, width, height,
                  edgecolor='k', facecolor='w',
                  fill=True,
                  text='',
                  loc=None,
-                 fontproperties=None
+                 fontproperties=None,
+                 visible_edges=None,
                  ):
 
         # Call base
         Rectangle.__init__(self, xy, width=width, height=height, fill=fill,
                            edgecolor=edgecolor, facecolor=facecolor)
         self.set_clip_on(False)
 
+        # set visible_edges
```

Comment:
This just re-states the code below and thus can be left out (essentially as all the comments in this function.